### PR TITLE
Integrate support for Coral 

### PR DESCRIPTION
**Pull request by jan-kiszka**
Add Google Coral packages to the example image by default. This increases our storage demand, so include the github running VM cleanup that @gylstorffq developed for #115.

### CODE REVIEW

**Comment by AsuraZeng** [comment]
using the previous integrated meta-coral on code.siemens.  there are some problems to run some demos.
and we get the information from google the problem is tf version is not correct.
link: https://github.com/google-coral/project-posenet/issues/81

Could  you pls check whether there are the same problems in this integration?

**Comment by jan-kiszka** [comment]
```
root@iot2050-debian:~/project-posenet# python3 simple_pose.py 
--2022-01-30 07:40:46--  https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/Hindu_marriage_ceremony_offering.jpg/640px-Hindu_marriage_ceremony_offering.jpg
Resolving upload.wikimedia.org (upload.wikimedia.org)... 91.198.174.208, 2620:0:862:ed1a::2:b
Connecting to upload.wikimedia.org (upload.wikimedia.org)|91.198.174.208|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 83328 (81K) [image/jpeg]
Saving to: ‘/tmp/couple.jpg’

/tmp/couple.jpg                                             100%[=========================================================================================================================================>]  81.38K  --.-KB/s    in 0.1s    

2022-01-30 07:40:47 (604 KB/s) - ‘/tmp/couple.jpg’ saved [83328/83328]

Inference time: 21 ms

Pose Score:  0.60673916
  NOSE                 x=210  y=152  score=1.0
  LEFT_EYE             x=223  y=138  score=1.0
  RIGHT_EYE            x=199  y=137  score=1.0
  LEFT_EAR             x=244  y=135  score=1.0
  RIGHT_EAR            x=183  y=129  score=0.8
  LEFT_SHOULDER        x=269  y=169  score=0.8
  RIGHT_SHOULDER       x=160  y=174  score=1.0
  LEFT_ELBOW           x=281  y=255  score=0.6
  RIGHT_ELBOW          x=153  y=253  score=0.9
  LEFT_WRIST           x=237  y=332  score=0.6
  RIGHT_WRIST          x=163  y=304  score=0.5
  LEFT_HIP             x=261  y=317  score=0.1
  RIGHT_HIP            x=173  y=309  score=0.1
  LEFT_KNEE            x=223  y=339  score=0.3
  RIGHT_KNEE           x=200  y=342  score=0.3
  LEFT_ANKLE           x=186  y=407  score=0.1
  RIGHT_ANKLE          x=188  y=410  score=0.2

Pose Score:  0.54908466
  NOSE                 x=400  y=145  score=1.0
  LEFT_EYE             x=418  y=129  score=1.0
  RIGHT_EYE            x=383  y=127  score=1.0
  LEFT_EAR             x=458  y=110  score=0.9
  RIGHT_EAR            x=371  y=121  score=0.2
  LEFT_SHOULDER        x=496  y=168  score=1.0
  RIGHT_SHOULDER       x=361  y=150  score=0.8
  LEFT_ELBOW           x=463  y=291  score=1.0
  RIGHT_ELBOW          x=331  y=245  score=0.8
  LEFT_WRIST           x=340  y=305  score=0.9
  RIGHT_WRIST          x=234  y=332  score=0.5
  LEFT_HIP             x=485  y=312  score=0.1
  RIGHT_HIP            x=364  y=327  score=0.1
  LEFT_KNEE            x=470  y=336  score=0.0
  RIGHT_KNEE           x=245  y=329  score=0.1
  LEFT_ANKLE           x=444  y=381  score=0.0
  RIGHT_ANKLE          x=188  y=410  score=0.2
```

Looks good to me. Any other issues should be reported to meta-coral.

One to-do we still have, but that can be done on top: cache the bazel fetch results in downloads so that users can rebuild in offline mode (https://github.com/siemens/meta-coral/issues/2).

File: Kconfig [hunk 1]
```diff
@@ -75,6 +75,18 @@ config KAS_INCLUDE_RT
...
+	default y
+	help
+	  Add driver and libraries to use a Google Coral Edge TPU accelerator
+	  card if it is installed in the IOT2050 extension slot.
+
+config KAS_INCLUDE_NO_CORAL
+	string
+	default "kas/opt/no-coral.yml"
+	depends on !CORAL
+
```
Comment (BaochengSu) [inline Kconfig:89]: Could this yaml be included automatically when CORAL != Y?

Comment (jan-kiszka) [reply Kconfig:89]: That is what the logic above is about.

File: recipes-core/images/iot2050-image-example.bb [hunk 2]
```diff
@@ -108,3 +108,11 @@ IMAGE_INSTALL += " \
     node-red-gpio \
     node-red-preinstalled-nodes \
     "
+
+IOT2050_CORAL_SUPPORT ?= "1"
+
+IMAGE_INSTALL += "${@ ' \
+    python3-pycoral \
```
Comment (AsuraZeng) [inline recipes-core/images/iot2050-image-example.bb:115]: do we know the version of these package?  seems there is no snapshot for the coral package.
could we lock the package version?

Comment (jan-kiszka) [reply recipes-core/images/iot2050-image-example.bb:115]: The locking is done via meta-coral, which we lock via kas. If you see an issue in version locking of meta-coral, please report there first.

**Review by BaochengSu — APPROVED** [review]

**Pull request merged**